### PR TITLE
[UI] fix the colors and difference in weather table

### PR DESCRIPTION
**Pull request by dpakach**
Fix the weather table to show the difference properly.

Hello @bartius-nigel, I just changed few UI elements to make the table look better. Please have a look and merge if you like. :+1: 

Before the changes:
Screenshot from 2020-12-03 12-21-37

After the changes:
Screenshot from 2020-12-03 12-21-11

### CODE REVIEW

**Comment by bartius-nigel** [comment]
I quite like that. Ill take a look at it later today (at work now)

File: src/components/Difference.svelte [hunk 1]
```diff
@@ -7,72 +7,77 @@
...
+  .invisible .difference {
     visibility: hidden;
   }
 
-  span {
+  .difference {
     position: relative;
     top: -10px;
     left: 5px;
-    font-size: 0.0000001em;
```
Comment (bartius-nigel) [inline src/components/Difference.svelte:75]: Can you change this to a different unit than em, i think the different browsers interpret this differently.

File: src/components/WeatherTable.svelte [hunk 2]
```diff
@@ -109,7 +107,7 @@
   tr:hover {
     color: royalblue;
   }
-  tr:nth-child(odd) {
+  tr {
```
Comment (bartius-nigel) [inline src/components/WeatherTable.svelte:110]: I like this, but could we change the background color to the white?

File: src/components/Difference.svelte [hunk 3]
```diff
@@ -7,72 +7,77 @@
...
 </script>
 
-<div>
-  {weather.highTempCelsius}
-  <span id="difference" class={differenceToStyleMap[difference]}>
-    {difference}
-  </span>
-</div>
+<td class="{getTempDifferenceKey(difference)} data-cell">
+  <div>
```
Comment (bartius-nigel) [inline src/components/Difference.svelte:32]: is it possible to just delete the div?

**Review by bartius-nigel — CHANGES_REQUESTED** [review]
This all looks really awesome. Thank you, just need some minor tweaks. Also, could you update the tests so that they pass? the CI wont let me merge this until that happens

File: src/components/Difference.svelte (hunk 1 above)
Comment (dpakach) [reply src/components/Difference.svelte:75]: I believe `em` and `rem` are based on the font size of the parent elements, so this shouldn't bring any inconsistency between browsers. I checked on firefox and chrome both looks ok

**Pull request merged**